src/PerpetualPositionRouter.sol

```diff
@@ -131,7 +159,7 @@ contract PerpetualPositionRouter {
         isExactInput: false,
         amount: amount,
         oppositeAmountBound: oppositeAmountBound,
-        deadline: type(uint256).max, // TODO: verify this is market order behavior, and do we need a
+        deadline: deadline, // TODO: verify this is market order behavior, and do we need a
```

Comment:
Is this still a TODO / if so is it tracked anywhere? Also seems the TODO comment is cutoff, "do we need a ..."

Reply:
https://github.com/ScopeLift/optimizoors-perpetual-protocol/pull/21/commits/e732a3f8a87d566881b80e64fa7209b4322142b1